spark/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -188,6 +167,27 @@ public StructType readSchema() {
     return lazyType();
   }
 
+  /**
+   * This is called in the Spark Driver when data is to be materialized into [ColumnarBatch]s
```

Comment:
Javadoc: Instead of `[...]`, did you mean to use either `{@link ...}` to link to the class, or `{@code ...}` to use fixed-width font?